src/javascript/app/ng-wig/ngWigEditableDirective.js

```diff
@@ -14,7 +14,15 @@ angular.module('ngWig')
         ngModelController.$setViewValue($element.html());
       }
 
-      $element.bind('blur keyup change paste focus click', function() {
+      if (angular.isFunction(scope.onPaste)) {
+        $element.on('paste', function(e) {
+          scope.onPaste(e).then(function(val) {
```

Comment:
actually it would be better to provide the event AND the current content.

This way we can not just replace the current content. We can ADD additional paste content as well